releasenotes/notes/baggage-support-be7eed26293f1216.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    tracing: Introduces support for Baggage as defined by the [OpenTelemetry specification](https://opentelemetry.io/docs/specs/otel/baggage/api/). Baggage is enabled by default. You can manually modify baggage using methods like ``context.set_baggage_item``, ``context.get_baggage_item``, ``context.get_all_baggage_items``, ``context.remove_baggage_item``, and ``context.remove_all_baggage_items``.
```

We need to format the link using rst:
```suggestion
    tracing: Introduces support for Baggage as defined by the `OpenTelemetry specification <https://opentelemetry.io/docs/specs/otel/baggage/api/>`_. 
```

Also we should simplify the release note. Any additional details about this feature should be added to our docs page.